webapp/src/components/simpleMentionEditor/simpleMentionEditor.tsx

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {
+    ReactElement,
+    useEffect,
+    useMemo,
+    useRef,
+    useState,
+} from 'react'
+import {getDefaultKeyBinding, EditorState, ContentState, DraftHandleValue} from 'draft-js'
+import Editor from '@draft-js-plugins/editor'
+import createMentionPlugin, {
+    defaultSuggestionsFilter,
+    MentionData,
+} from '@draft-js-plugins/mention'
+import '@draft-js-plugins/mention/lib/plugin.css'
+import './simpleMentionEditor.scss'
+
+import createEmojiPlugin from '@draft-js-plugins/emoji'
+import '@draft-js-plugins/emoji/lib/plugin.css'
+
+import {getWorkspaceUsersList} from '../../store/users'
+import {useAppSelector} from '../../store/hooks'
+import {IUser} from '../../user'
+
+const imageURLForUser = (window as any).Components?.imageURLForUser
+
+type Props = {
+    onChange?: (text: string) => void
+    onFocus?: () => void
+    onBlur?: (text: string) => void
+    initialText?: string
+    id?: string
+}
+
+const SimpleMentionEditor = (props: Props): ReactElement => {
```

Comment:
I don't like this name, it is just showing an implementation detail that is not more important that others. It should be name something like `MarkdownEditorInput` or something like that.

Reply:
Yes you are right.